ols-vms.conf

```diff
@@ -0,0 +1,11 @@
+vm.architecture = amd64
+vm.release = focal
+
+[serial-vault]
+# PPAs for python-codetree and charm-tools
+apt.sources = ppa:mojo-maintainers/ppa, ppa:juju/stable
```

Comment:
Since xenial there are no more deb packages for charm-tools in any PPA, this is now only available as snap.
Besides that, I don't see charm-tools being used by anything in serial-vault, you can probably drop the juju/stable ppa.

And a related issue: it's ok use focal? I see the current container in jenkaas is using xenial, will this break deployment on non-focal instances?

The current vms conf for the container is:
```
[serial-vault]
vm.class = lxd
vm.architecture = amd64
# Must be xenial because the thing is deployed on xenial and we live in the past.
vm.release = xenial
vm.update = True
vm.packages = git, python3-swiftclient, build-essential
jenkaas.secrets = swift/serial-vault:.config/swift/serial-vault
```
python3-swiftclient is required for the swift uploading bits, which is not required yet, but will not hurt to include it here, as it's a CI/Jenkins-only dependency.

Reply:
Did some clean-up. I would like to upload directly from Jenkins, so added here